TesseractCore/Value.swift

```diff
@@ -55,7 +55,7 @@ public enum Value: Printable {
 		case let Graph(graph):
 			return graph
 				.find { $1.isReturn }
-				.map { evaluate(graph, graph[$0].0, environment + (.Parameter(0, .Unit), argument)) }
+				.map { evaluate(graph, graph[$0].identifier, environment + (.Parameter(0, .Unit), argument)) }
```

Comment:
The sole fallout of removing the prior `DictionaryIndex<Identifier, T>` subscript from `Graph<T>`.